antarest/study/storage/rawstudy/model/filesystem/lazy_node.py

```diff
@@ -121,14 +139,24 @@ def save(self, data: Union[str, bytes, S], url: Optional[List[str]] = None) -> N
                 self.config.path.unlink()
             return None
 
-        self.dump(cast(S, data), url)
+        self.dump(t.cast(S, data), url)
         if self.get_link_path().exists():
             self.get_link_path().unlink()
         return None
 
+    def rename_filename(self, target: t.Union[Path, "LazyNode[t.Any, t.Any, t.Any]"]) -> None:
```

Comment:
I don't really like the repetition of `name`, I would prefer `rename_file`

Reply:
yes